test2code-agent/src/main/kotlin/com/epam/drill/plugins/test2code/Ast.kt

```diff
@@ -114,11 +122,13 @@ fun parseAstClass(className: String, classBytes: ByteArray): AstEntity {
 
 fun newAstClass(
     className: String,
-    methods: MutableList<AstMethod> = ArrayList()
+    methods: MutableList<AstMethod> = ArrayList(),
```

Comment:
No more reasons to use `newAstClass` as a function for creation the object of AstEntity class. Also no reasons to add any arguments to this function. I would delete this function and use constructor instead of.

Reply:
done.